demos/README.md

```diff
@@ -314,12 +326,12 @@ list above.
 ### Get Ready for Running the Demo Applications on Windows*
 
 Before running compiled binary files, make sure your application can find the Inference Engine and OpenCV libraries.
-Optionally download OpenCV community FFmpeg plugin. There is a downloader script in the OpenVINO package: `<INSTALL_DIR>\opencv\ffmpeg-download.ps1`.
+Optionally, download the OpenCV community FFmpeg plugin using the downloader script in the OpenVINO package: `<INSTALL_DIR>\extras\opencv\ffmpeg-download.ps1`.
```

Comment:
OpenCV is not optional but mandatory dependence for almost all OMZ demos, both for build and run

Reply:
This is about OpenCV FFmpeg plugin on windows. Instructions about OpenCV downloading is build section https://github.com/jkamelin/open_model_zoo/blob/jk/documentation/demos/README.md?plain=1#L93-L98. Before run we execute `setupvars`, assuming opencv is installed https://github.com/jkamelin/open_model_zoo/blob/jk/documentation/demos/README.md?plain=1#L328

Reply:
ok, then